.github/workflows/spack_cpu_build.yaml

```diff
@@ -14,17 +14,14 @@ jobs:
         # Minimal Build(s)
         # Need S3 mirror to have these builds speedup
         spack_spec:
-          # See #39 - ~python~mpi causes issues
-        # - exago@develop~mpi~ipopt~hiop~python~raja
-          # See #44 - +mpi~python should fail if no python in system,
-          #                       but the runner happens to have one...
-          - exago@develop+mpi~ipopt+hiop~python~raja ^openmpi
+          # See #39 - minimal build useful for sanity
+          - exago@develop~mpi~ipopt~hiop~python~raja
+          # See #16 / #44 - +python~mpi catches mpi4py / python config
+          - exago@develop~mpi~ipopt~hiop+python~raja
```

Comment:
Issue in `+mpi~python` should be equivalent to a build with `~mpi~python` as far as I can tell. Either way I think this is sufficient and we need to ensure these builds pass in future.